ads/google/a4a/utils.js

```diff
@@ -168,7 +168,6 @@ export function googleAdUrl(
         {name: 'brdim', value: additionalDimensions(win, viewportSize)},
         {name: 'isw', value: viewportSize.width},
         {name: 'ish', value: viewportSize.height},
-        {name: 'ea', value: '0'},
```

Comment:
Mentioned this in the top level comment. I noticed that we no don't send this parameter out in Delayed Fetch anymore, so for the sake of parity, I removed it here too.